src/redis-benchmark.c

```diff
@@ -1613,6 +1627,7 @@ int parseOptions(int argc, char **argv) {
 " -s <socket>        Server socket (overrides host and port)\n"
 " -a <password>      Password for Redis Auth\n"
 " --user <username>  Used to send ACL style 'AUTH username pass'. Needs -a.\n"
+"  --readpass <filename> Read password from file.\n"
```

Comment:
* There is an excessive space in the beginning of the --readpass line.

* Why not put -a and --readpass together? Now --user is between the two.

* "Needs -a" => "Needs -a or --readpass" (or maybe just "Needs password")

```suggestion
" -a <password>      Password for Redis Auth\n"
" --readpass <filename> Read password from file.\n"
" --user <username>  Used to send ACL style 'AUTH username pass'. Needs -a\n"
"                    or --readpass.\n"
```